### PR TITLE
Fix #1898 Made cookie a configurable plugin

### LINKED ISSUE #1898: Review cookie policy
Cookie policy popup should have a "read-more" link, maybe also an opt out to exit. 
The read more link should open the detailed privacy page, with the list of the used cookies.

### PR DESCRIPTION
**Pull request by MV88**
see #1898 

The cookie panel is visible if the flag in localeStorage is set to false.

To configure the plugn use 
```
"cfg":
{
  "externalCookieUrl" : "",
  "declineUrl" : "http://www.google.com"
}
```
if externalCookieUrl is set to an absolute path, by clicking on More details it will redirect there

### CODE REVIEW

File: web/client/components/cookie/Cookie.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                                bsStyle="primary" >
+                                <Message msgId="cookie.leave"/>
+                            </Button>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        ) : null;
+    }
+    moreDetails() {
```
Comment (mbarto) [inline web/client/components/cookie/Cookie.jsx]: Use arrow function for no-react methods (this one and accept) to allow autobinding

File: web/client/translations/data.de-DE [hunk 2]
```diff
@@ -54,6 +54,41 @@
             "tooltipDeactivate": "Beenden Sie den 3D-Modus",
             "tooltipActivate": "Geben Sie den 3D-Modus ein"
         },
+        "cookie":{
+            "info": "This website uses cookies to improve your experience. We'll assume you're ok with this, but you can opt-out if you wish.",
+            "moreDetailsButton": "More Details",
+            "leave": "Leave",
+            "accept": "Accept",
+            "moreDetails": {
```
Comment (mbarto) [inline web/client/translations/data.de-DE]: Not sure this is the best method to translate a long text content. Maybe having HTML is more flexible. Order of sentences can vary in different languages.

Comment (MV88) [inline web/client/translations/data.de-DE]: I saw i cannot put many tags inside a single translation. i.e. the whole page.  so i decided to put all the pieces. your suggestion require  page per language we will add, isn't it?

**Comment by offtherailz** [comment]
* Add third party cookies with linked informative (google, cloudflare)
* Add google and facebook to social
* Leave should leave the site, not open google in another tab
*

**Pull request merged**